polygon/erc20/tokens.sql

```diff
@@ -199,6 +199,1058 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
+\\x925a8aa33c970a7046dfdc8932adf32b8587dd41	xeth	18
+\\x13748d548d95d78a3c83fe3f32604b4796cffa23	KOGECOIN	9
+\\x02230382916b919d6d755a17b6f732528587e5f8	DIAMOND	18
+\\xcc081220542a60a8ea7963c4f53d522b503272c1	NFTY	18
+\\x1adcef5c780d8895ac77e6ee9239b4b3ecb76da2	TOTEM	6
+\\xc1b02e52e9512519edf99671931772e452fb4399	pMATIC	8
+\\x05d877764612e6011e141805b09c061c651547b3	IZZY	18
+\\xfa0116eb85e758f6cf12df9146076d79138d9f09	RUBY	18
+\\x84aef4af39160097da86d2d82437ea58f2798180	19D5	4
+\\x0982972408187de2cca7594d13c83f6c85988328	A3C5	5
+\\x6ef2a94ed8cc049d277f0f52eeb93e0bad933353	CLER	18
+\\x1e67124681b402064cd0abe8ed1b5c79d2e02f64	SLP	18
+\\x255707b70bf90aa112006e1b07b9aea6de021424	TETU	18
+\\xa0094a277e8191d2004161dda2df797e45208275	195F	18
+\\x81d5dcf097bc92f3c415b7b7da451a419643ef8f	E660	18
+\\x116ff0d1caa91a6b94276b3471f33dbeb52073e7	SLP	18
+\\x7594e01f6f8cc6f9133ca52c533bd036f5a68960	BFB7	18
+\\xd4024cd6acb71b1e729c406e1b8fd8fa863037d7	81D5	18
+\\x3439fef175fa648c280f577a651ce2f291fb913e	A5Ad	18
+\\x9af3b7dc29d3c4b1a5731408b6a9656fa7ac3b72	PUSD	18
+\\x262241f4dc85c1ae38ff3611203e7b3b687b4d46	6E40	18
+\\x6dad9f118543605517a5e79e6b08a8d133fea291	A1A8	18
+\\x88f55a7f2211806797a2edd081c27504c6f26e22	RX	6
+\\xdd7859ae450e1600e63f068dae21df8f84f54a44	3F3B	18
+\\x190d9ca444b619c572fbcbed40df3eef98b9849c	FF97	18
+\\x434b769a741026a706b7bcbdb0d23107a86671be	18F1	18
+\\xaaf96779dde695246d0f0ba60d9a4308f0b97b73	FC4F	18
+\\x12bc596adcc84c384d108b0de32eecf5b8f9d1bd	8B61	18
+\\x40662b53f6f4d9b6e28929ebac9321190c352a2f	034E	18
+\\xa88c5693c9c2549a75acd2b44f052f6a5568e918	YDZ	5
+\\x3e4b6b5626cf2b80b219f3405076323fcf3cdba2	02CA	18
+\\x980111ae1b84e50222c8843e3a7a038f36fecd2b	STACK	18
+\\xb29f7077d80cc45438bc05eb867d8c3565c8f761	808A	18
+\\xfc0a985179c80d9d38d7acefad60905eaa7059b4	BC94	18
+\\xc1742019a351e3430e4bb90cba7477588703487b	AB69	18
+\\x08e175a1eac9744a0f1ccaeb8f669af6a2bda3ce	E8	9
+\\x9a575498ce240fe43504f53d68eef6440f0cf280	APENK	18
+\\x4b56708bca811ed5851b4e41e99d36bfccbacb9f	PLF	18
+\\xa8dc64ca15d66f6509af7112dbc981ed8037681e	EB7E	18
+\\xd8a7fae6dcc6fbad7c2d7eadd5752294fcd07611	12C0	18
+\\x23c70dd93d4ecac7fb93631488c5412e02f4a57c	REIN	18
+\\x1a4ad39a4cadbefebbf6a621e72a6ca539afbaa0	2C5B	18
+\\x8c6b401a5e322bb437be419f322597d1833c9d69	01B0	18
+\\x4a59226302417a028145675d6ed96b38400025bd	45F9	18
+\\xaecebfcf604ad245eaf0d5bd68459c3a7a6399c2	RAMP	18
+\\x93803af609a9d6d492968f8a043fb950abfbf5d1	06E2	18
+\\x1c2e7118591ec1b6122e4fe3d645706a9747b10e	ESPRO	18
+\\x2e502bf17e29fe9d59f616501a1d189e93fc676f	808TA	18
+\\x695fc8b80f344411f34bdbcb4e621aa69ada384b	NITRO	18
+\\x32934cb16da43fd661116468c1b225fc26cf9a8c	SNE	18
+\\x6b7e88b6c6051c03ec57a5e78b880999633bd047	OIL	18
+\\x9fb758d4243f016ad049c412076cfffda22c9d06	REIN	18
+\\xe13ddcdc89b7eb61eabdd90e80c1e7f888769ad9	BEASTVERSE	9
+\\x14a92603d4f4bf605e3ef206f7614a0ab4248597	FET	8
+\\xfb8a07e99450d1dc566da18a8f0e630addefdd3e	STN	18
+\\x1fd6cf265fd3428f655378a803658942095b4c4e	YELD	V2	18
```

Comment:
i fixed this row locally, but don't have access to push to your branch. please put regular space between 'YELD' and 'V2' as it's meant to be on column

Reply:
OK, it's fixed, thanks